web/developer.actyx.com/docs/how-to/app-auth/generate-app-license.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Generate your own app license
+id: generate-app-license
+hide_title: false
+hide_table_of_contents: false
+image: /images/defaults/default.svg
+---
+
+This guide will walk you through the process of licensing an app.
+
+When running an Actyx node in production mode, your apps are required to be licensed.
+This mechanism provides a more fine grained control over which apps are _authorized_ to run on which nodes.
+
+To generate an app license, we use `ax-cert`:
+
+```bash
+$ cd rust/actyx
+$ cargo run --bin ax-cert -- app-license \
+    --actyx-private-key 0WBFFicIHbivRZXAlO7tPs7rCX6s7u2OIMJ2mx9nwg0w= \
```

Comment:
Add a sentence informing that the private key is acquired during compiling, and link the relevant docs page.